Games/Niko: Through The Dream/Steam/script.json

```diff
@@ -0,0 +1,7 @@
+{
+	"scriptName": "Steam",
+	"compatibleOperatingSystems": ["MACOSX", "LINUX"],
+	"testingOperatingSystems": ["LINUX"],
```

Comment:
It means that your program cannot work without patching the original protection. It may be illegal to install it depending on your country